src/go/rpk/pkg/cli/cmd/generate/grafana.go

```diff
@@ -809,3 +850,18 @@ func htmlHeader(str string) string {
 		str,
 	)
 }
+
+const dashboardHelp = `You can select one of the following dashboard types:
+
+  consumer-metrics:  Allows for monitoring of Java Kafka consumers, using the 
```

Comment:
Can we have use cobra's ValidArgsFunction or ValidArgs to suggest these on tab?

Reply:
or maybe it's a flag completion

Reply:
Both ValidArgs and ValidArgsFunction are for non-flag arguments:

```
// ValidArgs is list of all valid non-flag arguments that are accepted in shell completions
// ValidArgsFunction is an optional function that provides valid non-flag arguments for shell completion.
```
There is no way atm to validate flags nor restrict the flag values to a specific set of values in pflag, see https://github.com/spf13/pflag/issues/236

What do you mean by: 

> or maybe it's a flag completion 

?

Reply:
I added flag completion in the last commit.